### PR TITLE
automate stats update 

### PR DESCRIPTION
**Pull request by 0oM4R**
### Description

make the stats image calls the stats update endpoint each 6h to update stats 

### Changes

- add zinit 
- add cron zinitt process 
- add ngnix zint process
- add cron job to call localhost/api/status-update each 6h 

### Related Issues

- #2493 

### Documentation PR

For UI changes, Plaese provide the Documetation PR on [info_grid](https://github.com/threefoldtech/info_grid)

### Checklist

- [ ] Tests included
- [x] Build pass
- [ ] Documentation
- [x] Code format and docstrings
- [ ] Screenshots/Video attached (needed for UI changes)

### CODE REVIEW

File: packages/stats/Dockerfile [hunk 1]
```diff
@@ -11,3 +11,11 @@ RUN yarn lerna run build --no-private && yarn workspace @threefold/stats build
 FROM nginx:1.25.3-alpine
 COPY --from=build /app/packages/stats/dist /usr/share/nginx/html
 COPY --from=build /app/packages/stats/nginx /etc/nginx
+RUN mkdir -p /etc/zinit;
+COPY --from=build /app/packages/stats/zinit /etc/zinit
+
+RUN curl --location  https://github.com/threefoldtech/zinit/releases/download/v0.2.10/zinit -o /sbin/zinit && \
```
Comment (Mahmoud-Emad) [inline packages/stats/Dockerfile:17]: I noticed that the version [v0.2.14](https://github.com/threefoldtech/zinit/releases/tag/v0.2.14) is released, I'm not sure if we have to use it instead

**Review by Mahmoud-Emad — APPROVED** [review]
Good job ya Omar, I'm wondering if we need to update just the Zinit version

**Review by amiraabouhadid — COMMENTED** [review]
Is there a reason why you left out the endpoint `https://stats.dev.grid.tf/api/stats-summary` mentioned in the issue?

**Comment by AhmedHanafy725** [comment]
> Is there a reason why you left out the endpoint `https://stats.dev.grid.tf/api/stats-summary` mentioned in the issue?

no, just as an example. The workflow solution wasn't the optimal solution to update the stats as we have many stacks per network so the update should happen within the image itself.

**Review by amiraabouhadid — APPROVED** [review]

**Pull request merged**